cleverpush/src/main/AndroidManifest.xml

```diff
@@ -12,7 +12,9 @@
         <activity android:name=".stories.StoryDetailActivity"
             android:theme="@style/Theme.AppCompat.NoActionBar" />
 
-        <activity android:name=".banner.WebViewActivity" />
+        <activity android:name=".banner.WebViewActivity"
```

Comment:
Why does the WebViewActivity live under `banner ` package? In this case we use it for banners + for notifcations.
Please move it to somewhere else.